scipy/stats/tests/test_stats.py

```diff
@@ -2875,19 +2883,19 @@ def test_ttest_rel():
     y = (stats.norm.rvs(loc=5, scale=10, size=501) +
          stats.norm.rvs(scale=0.2, size=501))
     y[500] = np.nan
-    with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', category=RuntimeWarning)
+
+    with np.errstate(invalid="ignore"):
```

Comment:
Hmmm, I am wondering if I should just put the `np.errstate` into the module setup for this file and remove them all? What do you think?